engli/dunia_baxe.md

```diff
@@ -387,13 +387,13 @@ form that sounds familiar to speakers of as many languages as possible.
 | Pandunia root | English    | Portuguese | Spanish    | French     | German     | Russian    |
 |---------------|------------|------------|------------|------------|------------|------------|
 | **kort**      | short      | curto      | corto      | court      | kurz       | korotkiy   |
-| **now**       | new        | novo       | nuevo      | nouveau    | neu        | novıy      |
+| **nov**       | new        | novo       | nuevo      | nouveau    | neu        | novıy      |
 | **marc**      | march      | marcha     | marcha     | marche     | Marsch     | marš       |
 | **post**      | post (mail)| (postal)   | posta      | poste      | Post       | počta      |
 
 The suffixes of Pandunia are applied on the roots. Here are some
 resulting Pandunia words: korti (short), korte (shorty), korta 
-(shorten), nowi (new), nowe (news), nowa (renovate).
+(shorten), nov (new), nove (news), nova (renovate).
```

Comment:
novi. Adjectives normally include -i and I don't see why "nov" would be an exception anymore.

Reply:
Good catch!